src/server/pps/pretty/pretty.go

```diff
@@ -41,20 +43,55 @@ func PrintPipelineInfo(w io.Writer, pipelineInfo *ppsclient.PipelineInfo) {
 		}
 	}
 	if pipelineInfo.OutputRepo != nil {
-		fmt.Fprintf(w, "%s\t\n", pipelineInfo.OutputRepo.Name)
+		fmt.Fprintf(w, "%s\t", pipelineInfo.OutputRepo.Name)
 	} else {
-		fmt.Fprintf(w, "\t\n")
+		fmt.Fprintf(w, "\t")
 	}
+	fmt.Fprintf(w, "%s\t\n", pipelineState(pipelineInfo))
+}
+
+func PrintDetailedPipelineInfo(pipelineInfo *ppsclient.PipelineInfo) {
+	bytes, err := json.MarshalIndent(pipelineInfo, "", "    ")
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+
+	obj, err := gabs.ParseJSON(bytes)
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+
+	// state is an integer; we want to print a string
+	_, err = obj.Set(ppsclient.PipelineState_name[int32(pipelineInfo.State)], "state")
+	if err != nil {
+		fmt.Println(err.Error())
+	}
+
+	fmt.Println(obj.StringIndent("", "    "))
 }
```

Comment:
I think long term we're going to want a `PrintDetailed____Info` for every object we have and it should probably be a little bit prettier than just printing the JSON. This is fine for now though.